astro/src/content/docs/apis/index.mdx

```diff
@@ -183,6 +183,10 @@ You can explore our APIs using a self hosted instance, one you run yourself on a
 
 You can use our [API explorer](/docs/apis/api-explorer) or our [Postman collection](https://www.postman.com/fusionauth).
 
+## Backwards Compatibility
+
+FusionAuth strives to make no backwards incompatible changes to our APIs and features. Occasionally, FusionAuth will deprecate APIs or features in preparation for removal. Please see our [Deprecation Policy](/docs/operate/roadmap/deprecation-policy) for more.
```

Comment:
I think this would read better without the negative. "make no" is hard to read. 

Suggestion:

> FusionAuth strives to maintain backwards compatibility when making changes to APIs and features. Occasionally, ....